packages/forms/stories-core/layouts.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { HeaderBar, Layout, Dialog, Drawer } from '@talend/react-components';
+import { action } from '@storybook/addon-actions';
+
 import Form from '../src/index';
 
 const simple = require('./json/concepts/core-simple.json');
+const old = require('../stories/json/simple.json');
+
+const actions = [
+	{
+		name: 'cancel',
+		style: 'link',
```

Comment:
I think this should be `inverse` & `default`? (it affects alignment if we use link style)